app.py

```diff
@@ -20,7 +20,49 @@ def upload_and_predict():
         return jsonify({'message': 'File uploaded successfully'}), 200
     return jsonify({'message': 'An error occurred'}), 422
 
+@app.route("/predict", methods=["POST"])
+def predict():
+    metaname =  "/Users/saurabh7/Downloads/model.meta"
```

Comment:
the model names are relative to your local directory
can you move those to the project directory and use that